packages/mds-agency/api.ts

```diff
@@ -74,9 +74,9 @@ function api(app: express.Express): express.Express {
           return res.status(401).send({ error: 'authentication_error', error_description: 'Unauthorized' })
         }
       }
-    } catch (err) {
+    } catch (error) {
       /* istanbul ignore next */
-      logger.error(req.originalUrl, 'request validation fail:', err.stack)
+      logger.error('request validation fail:', { originalUrl: req.originalUrl, error })
```

Comment:
```suggestion
      logger.error('request validation fail:', { originalUrl: req.originalUrl, error: error.stack })
```
`error.toString()` doesn't include the stack